birdhouse/scripts/get-components-json.include.sh

```diff
@@ -57,7 +57,7 @@ BIRDHOUSE_DEPLOY_COMPONENTS_BASE="bird-house/birdhouse-deploy:"
 BIRDHOUSE_DEPLOY_COMPONENTS_LIST=$( \
   echo "${BIRDHOUSE_DEPLOY_COMPONENTS_LIST_ACTIVE}" \
   | grep "${BIRDHOUSE_DEPLOY_COMPONENTS_LIST_KNOWN}" \
-  | sed -E 's|^\s*([A-Za-z0-0./_-]+)\s*$|"\1",|g' \
+  | sed -E 's|^\s*([A-Za-z0-9./_-]+)\s*$|"\1",|g' \
```

Comment:
Oh geez !!!  But then how on earth did this work when `./pavics-compose.sh` is called manually and not when it is called by autodeploy !

Reply:
All the services except `ncWMS2` are alpha-only, not alphanumeric!  Could it be that `autodeploy` does not use it?
Silly mistake. 😅 Nice find.